cen/events.py

```diff
@@ -71,14 +71,12 @@ def send_email_notification(self, request):
         try:
             template_source = get_rule_product(installation_id, product_id).message
         except peewee.DoesNotExist:
-            self.logger.info(f'No template found for product {product_id} and {installation_id}')
-            return BackgroundResponse.fail('No template found')
+            return BackgroundResponse.done()
 
         try:
             body = markdown.markdown(jinja.render(template_source, request))
-        except Exception as e:
-            self.logger.info(f'Error in template: {e}')
-            return BackgroundResponse.fail(str(e))
+        except Exception:
+            return BackgroundResponse.done()
```

Comment:
why to remove the logger?? Reponse must be done() but logging must be there

Reply:
Done